include/common/stack_string.h

```diff
@@ -0,0 +1,173 @@
+//
+//  stack_string.h
+//
+//  Created by Oleksandr Kysil on 28.02.2021.
+//
+
+#ifndef MAX_CHARACTER
+#define MAX_CHARACTER 256
+#endif
+
+#ifndef stack_string_h
+#define stack_string_h
+
+class stack_string {
+public:
+    const char* c_str() {
+        return m_buffer;
+    }
+    
+    /*
+        Constructors
+     */
+    
+    stack_string() {}
```

Comment:
check buffers content if object was constructed using this constructor. it might be full of garbage and we don't need that. better set 1st element to \0